### PR TITLE
wallabag 2.2.0

### PR DESCRIPTION
**Pull request by j0k3r**
:construction: This is just a follow up for the next 2.2.0 release.
- Delete user #2177 
- API: ability to export entry in all available format (epub, pdf, etc...) #2372 #2453
- Use new tcpdf library version #2456 
- Bring annotations to API #1800 
- Reset account #2401
- Importing articles with invalid UTF-8 #2468
- Added QRCode and link to configure android application #2523
- Added relation between API Client and User #2499
- Download pictures #2180
- Use created_at as default sort #2535
- Removed Maintenance bundle #2551
- Add Pinboard import #2548
- Added new tag form in mobile view for Material theme #2553
- Added unmark.it sharing #2566
- Added shortcuts #2495
- Added a configuration to define the redirection after archiving an entry #2547
- wallabag can’t work on PostgreSQL <= 9.1 #2604
- Added a simple search engine #2543
- RSS paging #2609
- Add ability to use Redis with password #2621
- Add white-text class to card-title too #2630
- Add ability to reload entry from API #2614
- Fix card action overlap #2613 
- Added authentication for restricted access articles #2317
- Fix password layout #2620
- Material UI Changes #2622
- Enforce lowercase on domain name filter #2654
- Fix incorrect reading time in non-latin languages #2659
- Add list view #2662
- Added hasColumn() in migration to check column existence #2650
- Removed FOSUser attributes removed in alpha4 #2663
- Added floating button on entry view for mobile #2670
- Upgrade to Symfony 3.2 #2667
- Added QoS for RabbitMQ #2690
- Removed outputWalkers for pagination #2701
- Run migration on each test #2696
- Share to Shaarli - Include tags #2725
- Fix double overlay on expanded right sidenav #2729
- Fix right side-nav scrolling on low height window #2728
- Avoid false preview image #2742

### CODE REVIEW

File: docs/de/user/import.rst [hunk 1]
```diff
@@ -42,6 +42,7 @@ Du musst wallabag erlauben, mit deinem Pocketaccount zu interagieren.
 Deine Daten werden importiert. Datenimport kann ein sehr anspruchsvoller Prozess für deinen Server
 sein (wir müssen daran arbeiten, um diesen Import zu verbessern).
 
+<<<<<<< HEAD
```
Comment (tcitworld) [inline docs/de/user/import.rst]: Raté

**Comment by j0k3r** [comment]
:man_singer:

**Comment by nicosomb** [comment]
wow :') 🎉

**Comment by tcitworld** [comment]
📈

**Pull request merged**